test/fuzz/readKeyArmored.js

```diff
@@ -0,0 +1,28 @@
+import { FuzzedDataProvider } from '@jazzer.js/core';
+
+import openpgp from '../initOpenpgp.js';
+
+const ignored = ['Misformed armored text'];
+const MAX_MESSAGE_LENGTH = 4096;
+
+function ignoredError(error) {
+  return ignored.some(message => error.message.includes(message));
+}
+
+/**
+ * @param { Buffer } inputData
+ */
+export function fuzz (inputData) {
+
+  const data = new FuzzedDataProvider(inputData);
+  const fuzzedText = data.consumeString(MAX_MESSAGE_LENGTH, 'utf-8');
+  const armoredKey = `-----BEGIN PGP PRIVATE KEY BLOCK-----\n ${fuzzedText} -----END PGP PRIVATE KEY BLOCK-----`;
+
+  return openpgp.readKey({ armoredKey })
```

Comment:
Most (if not all) of the functions being tested in this PR are actually async. I just want to make sure that the fuzzing lib is `await`ing them..? Since it's not being done explicitly in this code (nor is the function declared `async`) 🙂

Reply:
It does.
Whenever jazzer.js detects that the fuzz target returns a promise it will await the completion before it calls the fuzz target again.
You can https://youtu.be/KyIhxEiNnfc?t=1571